data/remnant/remnant missions.txt

```diff
@@ -970,6 +970,140 @@ mission "Remnant: Learn Sign Follow Up"
+				goto terminuschoice
+			label secrets
+			`	Thinking back, you recall how you helped Ivan and a team of researchers from the Deep study the Ember Waste threshold wormhole. Research like that might be of interest to the Remnant, seeing as it could open the door to research vessels venturing into the Ember Wastes.`
+			label terminuschoice
+			choice
+				`	(Find a prefect.)`
+				`	(Forget about it.)`
+					decline
+			`	For lack of a better place to go, you stop by the information desk where you picked up the sign language course. You tell the attendant that you have something potentially important - but non-urgent - to discuss in regards to Remnant security. They tap a few buttons, then give you directions down a hallway. You quickly find yourself in an armory where several Remnant guards are in the process of cleaning their weapons. One stands up and comes over to you.`
+			`	"Greetings Captain, I am Prefect Torza. I was told you had some information that could be important?" He takes a seat on a bench on the far side of the room from the other Remnant, and gestures for you to take a seat with him.`
+			branch confirmation
+				has "deep: did not reveal remnant"
+			`	You give him some background about the current state of human and Hai understanding about the wormholes - or the lack thereof - then explain how you met a team of researchers who were using drones to study the wormhole exit in the Terminus system. He quickly picks up on the potential threat if someone figures out the key to using them, especially since you have demonstrated that it only takes one curious human to figure it out.`
+			goto investigatechoice
+			label confirmation
+			`	You give him some background about the current state of human and Hai understanding about the wormholes - or the lack thereof - then explain how you met a team of researchers who were using drones to study the wormhole exit in the Terminus system and now know that quantum keystones can stabilize it. He quickly picks up on the potential threat that this development represents.`
+			`	Thinking carefully, he asks, "It is clear that we need to keep an eye on this situation. You know more about the rest of the galaxy than we do, so could you find them and see how much progress they have made?"`
+			choice
+				`	"I'd be happy to."`
+				`	"I'd prefer not to."`
+					decline
+			`	"Excellent. See what you can find out about their research and results, but it would probably be wise to avoid making direct contact with them."`
+				accept
+	on complete
+		conversation
+			`Arriving on Bounty, you take a discreet look around the spaceport, but you don't find the researchers or any sign of them. It appears that they packed up and left since the last time you saw them. Time to look around for some info.`
+
+
+
+mission "Remnant: Follow-up on Terminus Researchers A-2"
+	name "Check up on Terminus Researchers"
+	description "Tracking down the wormhole researchers."
```

Comment:
This might need to be more informative about what to do to complete the mission.